purchases/src/main/kotlin/com/revenuecat/purchases/paywalls/components/Localization.kt

```diff
@@ -16,7 +16,9 @@ import kotlinx.serialization.encoding.Encoder
 @JvmInline
 internal value class LocaleId(val value: String)
 
-internal typealias LocalizationKey = String
+@Serializable
+@JvmInline
+internal value class LocalizationKey(val value: String)
```

Comment:
I totally know what a `@JvmInline` is... yup yup yup 😅

Reply:
Haha, this instructs the Kotlin compiler to make it a [`value class`](https://openjdk.org/projects/valhalla/) from a Java perspective.